lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -110,6 +110,67 @@ class ConvertAtenBinaryOp : public OpConversionPattern<AtenOpT> {
   }
 };
 
+// FIXME(AG): This will eventually go into a Tosa*Utils file
+LogicalResult torchScalarToTosaTensor(ConversionPatternRewriter &rewriter,
+                                      Operation *op, Value torchScalarValue,
+                                      Value &tosaTensor, Type dtype) {
+  if (dtype.isa<mlir::FloatType>()) {
+    double scalarValue;
+
+    if (!matchPattern(torchScalarValue, m_TorchConstantFloat(&scalarValue)))
+      return failure();
+
+    tosaTensor =
+        mlir::tosa::getTosaConstTensorSingleF32(rewriter, op, scalarValue);
+  } else if (auto intTyp = dtype.dyn_cast<mlir::IntegerType>()) {
+    int64_t scalarValue;
+
+    if (!matchPattern(torchScalarValue, m_TorchConstantInt(&scalarValue)))
+      return failure();
+
+    auto w = intTyp.getWidth();
+    if (w != 32 && w != 64)
+      return op->emitError("Unsupported integer type");
+
+    if (w == 32) {
+      tosaTensor = tosa::getConstTensor<int32_t>(
+                       rewriter, op, {static_cast<int32_t>(scalarValue)}, {})
+                       .getValue();
+    } else if (w == 64) {
+      tosaTensor =
+          tosa::getConstTensor<int64_t>(rewriter, op, {scalarValue}, {})
+              .getValue();
+    }
+    return success();
+  } else
+    return op->emitError("Usupported element type");
+
+  return success();
+}
+
+LogicalResult torchAlphaToTosaTensor(ConversionPatternRewriter &rewriter,
+                                     Operation *op, Value alphaScalar,
+                                     Value &alphaTensor, Type dtype,
+                                     bool checkForUnity) {
+  if (succeeded(torchScalarToTosaTensor(rewriter, op, alphaScalar, alphaTensor,
+                                        dtype)))
+    return success();
+
+  // alpha has not been specified
```

Comment:
please use full english prose/capitalization per LLVM coding standards https://llvm.org/docs/CodingStandards.html#commenting. I would suggest adding a period and enclosing alpha with preformatted text.

```
// `alpha` has not been specified.
```

(you don't always need to use backticks for variable names, but since it starts the sentence, it makes it clear that it doesn't need to be capitalized)

I've added review comments for a few other cases, but please audit the comments throughout to make sure they comply.